R/render-plot.R

```diff
@@ -106,164 +89,178 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
   session <- NULL
   outputName <- NULL
 
+  # Calls drawPlot, invoking the user-provided `func` (which may or may not
+  # return a promise). The idea is that the (cached) return value from this
+  # reactive can be used for varying width/heights, as it includes the
+  # displaylist, which is resolution independent.
+  drawReactive <- reactive({
```

Comment:
The old `plotObj` reactive had `label = "plotObj"`. Should we keep it here? I think it helped profiles to look nice, but I can't remember for sure if it helped in other ways.

Reply:
Good idea, I didn't mean to drop that.